### ISSUE TITLE
DIscord rich presence is still active even when it's off

### ISSUE THREAD

**Issue by Faroras** [homchom/recode#10]
- I have discord rich presence turned off
- i sometimes get random /locate messages in chat even though i didn't do /locate
- when i do /locate i don't see any message most of the time (probably gets hidden)

**Comment by deflatedsheep** [comment]
I was talking to homchom about the /locate spam, and yeah it does use /locate even when the presence is off. I think it got slowed down so it doesn't spam as fast.

**Comment by homchom** [comment]
Yeah, I plan on fixing this properly in the next beta. The mod definitely shouldn't run /locate on loop.

**Comment by Electric131** [comment]
For the record, I have fixed the /locate looping so this shouldn't happen again if it is put in place.
